firmware/Sequencer.h

```diff
@@ -85,13 +94,6 @@ static void sequencer_init() {
   double_speed = false;
 }
 
-void sequencer_restart() {
-  MIDI.sendRealTime(midi::Start);
-  usbMIDI.sendRealTime(midi::Start);
-  delay(1);
```

Comment:
the delay here is deliberate, to give receiving devices a chance to get ready for the first clock. We might be able to get rid of it or shorten it but that needs some more testing

Reply:
Yeah, I understand the intention, but the reason I removed it is because I don't think it has the intended effect here.
We can add it here again, but to properly make sure we're not spamming the MIDI output we should handle it where we do the final MIDI sending. Maybe as part of https://github.com/datomusic/duo-imxrt/issues/156

Reply:
Delay added again in https://github.com/datomusic/duo-imxrt/pull/155/commits/2432d2e18e18a9a8dc85f15fe77cdaac27ad2b05